src/main/java/com/suse/saltstack/netapi/parser/JsonParser.java

```diff
@@ -268,4 +315,28 @@ public StartTime read(JsonReader jsonReader) throws IOException {
             return new StartTime(subStr);
         }
     }
+
+
+    /**
+     *
+     */
```

Comment:
Please either remove this empty comment or add some documentation to this class.